### PR TITLE
Revise README

### PR DESCRIPTION
**Pull request by marcdavi-es**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,101 +2,105 @@
...
 ```
 
-CAUTION: this library will work only with iterable properties of the state, created by [immutable](https://github.com/facebook/immutable-js/).
+at the target location.
+
+If you want to specify different locations for these three, pass an object like
+
+```js
+{
+  request: (Array<string>),
```
Comment (bratushka) [inline README.md]: Array can contain not only strings, but numbers and even booleans or objects afaik

File: README.md [hunk 2]
```diff
@@ -105,62 +109,31 @@ dispatch(fromTo(
...
-    badBoy: {
+    goodBoy: {
       isRequesting: true,
     },
   }),
 }
 ```
-State after the axios call:
+
+State if `from` rejects
```
Comment (bratushka) [inline README.md:135]: badBoy -> goodBoy

**Review by bratushka — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (marcdavi-es) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (marcdavi-es) [reply README.md:135]: Fixed.

**Review by bratushka — APPROVED** [review]

**Pull request merged**